### DISCUSSION TITLE
定期的に自動更新される記事からのリンク通知を停止したい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#563] [要望・アイデア・不具合]
いいね数ランキングを定期的に自動更新している記事があるのですが、記事を書くたびに毎回のようにリンクされて「またか・・・」という気持ちになっています（画像赤枠部分）。

<img width="776" alt="q" src="https://github.com/increments/qiita-discussions/assets/1148320/82992bf9-cbb5-4b40-9ba0-f86b22ca87f2">

当該記事のユーザーはミュートしたのですが、ミュートしても通知はまだ有効になっていました。
こういう場合はブロックするといいのでしょうか？
ヘルプを見てもよくわかりませんでした。

https://help.qiita.com/ja/articles/qiita-mute
https://help.qiita.com/ja/articles/qiita-block

とはいえ、できればユーザー単位でミュート・ブロックするより、記事単位で通知をミュートできるようになっている方がありがたいです。
そのユーザーさんが自動更新ではない手書き記事でふつうにリンクしてくる場合は、通知されてもよいと思うので。

ご確認・ご検討よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご不便をおかけしており申し訳ありません🙇

記事単位で通知をミュートできるようにする事は、実装コストと提供できる価値を考えると、現時点での優先度は低めになりそうです🙇

ただ、現在の仕様ではブロック（またはミュート）しても、通知は送られるようになっているので、こちらの改善は検討したいと思います。

現時点で可能な対策としては、全てのリンクされた際の通知がオフになってしまいますが、設定の「メールアドレスと通知」で通知をオフにしていただくことも可能です。
（自動更新ではない記事からのリンク通知は受け取りたいかと思うのですが、現時点ではこれしか方法がなく申し訳ありません🙏）
https://qiita.com/settings/notifications

![image](https://github.com/increments/qiita-discussions/assets/103916185/40d01047-dfba-4d20-abd7-3077805054ad)

**Reply by JunichiIto** [reply]
返信ありがとうございます。

> 記事単位で通知をミュートできるようにする事は、実装コストと提供できる価値を考えると、現時点での優先度は低めになりそうです🙇
ただ、現在の仕様ではブロック（またはミュート）しても、通知は送られるようになっているので、こちらの改善は検討したいと思います。

了解しました。とりあえずミュートで通知が来なくなるようになればまだよいかな〜という感じです。
ご検討よろしくお願いします。

> 現時点で可能な対策としては、全てのリンクされた際の通知がオフになってしまいますが、設定の「メールアドレスと通知」で通知をオフにしていただくことも可能です。
（自動更新ではない記事からのリンク通知は受け取りたいかと思うのですが、現時点ではこれしか方法がなく申し訳ありません🙏）

なるほど。ただ、「自動更新ではない記事からのリンク通知は受け取りたい」のでこの設定はまだ有効なままにしておきます。

**Reply by yuta0709** [reply]
@JunichiIto 
大変お待たせいたしました🙇‍♂️

ミュートまたはブロックしているユーザーからの通知が届かないようにアップデートいたしました。
https://qiita.com/release-notes/2025-11-10

この度は貴重なご意見をお寄せいただきありがとうございました。
今後とも Qiita をよろしくお願いいたします🙏

**Reply by JunichiIto** [reply]
@yuta0709 おお、ご対応どうもありがとうございます！！了解しました！😆
